runners/spark/src/test/java/org/apache/beam/runners/spark/SparkRunnerDebuggerTest.java

```diff
@@ -81,14 +81,14 @@ public void debugBatchPipeline() {
         "sparkContext.parallelize(Arrays.asList(...))\n"
             + "_.mapPartitions("
             + "new org.apache.beam.runners.spark.examples.WordCount$ExtractWordsFn())\n"
-            + "_.mapPartitions(new org.apache.beam.sdk.transforms.Contextful())\n"
+            + "_.mapPartitions(new org.apache.beam.sdk.transforms.MapperBase$1())\n"
```

Comment:
It's not clear to me why this PR makes this show up as `MapperBase$1` rather than `Contextful`. Is this potentially pointing to a scope leak causing more to get serialized than necessary?